packages/edit-site/src/components/template-details/index.js

```diff
@@ -2,24 +2,35 @@
  * WordPress dependencies
  */
 import { __, sprintf } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { getTemplateInfo } from '../../utils';
 
-export default function TemplateDetails( { template } ) {
+export default function TemplateDetails( { template, onClose } ) {
+	const { openNavigationPanelToMenu } = useDispatch( 'core/edit-site' );
 	if ( ! template ) {
 		return null;
 	}
+
 	const { title, description } = getTemplateInfo( template );
 
+	const showTemplateInSidebar = () => {
+		onClose();
+		openNavigationPanelToMenu( 'templates' );
+	};
+
 	return (
 		<div className="edit-site-template-details">
 			<p className="edit-site-template-details__heading">
 				{ __( 'Template details' ) }
 			</p>
-
+			<Button onClick={ showTemplateInSidebar } isTertiary>
+				{ __( 'View in navigation.' ) }
```

Comment:
I think this could use a translation context, or maybe even a reword (e.g. "View in navigation _sidebar_", which doesn't sound as good, but it's more clear). 🤔

Reply:
good idea. for what it's worth, I didn't spend any time on this button other than wiring it up, so it currently looks very out of place. mostly just for testing the redux changes